janusgraph-dist/pom.xml

```diff
@@ -684,29 +690,22 @@
                         <executions>
                             <execution>
                                 <id>assemble-full-distribution-archive</id>
-                                <phase>package</phase>
+                                <phase>${build.full}</phase>
                                 <goals>
                                     <goal>single</goal>
                                 </goals>
                                 <configuration>
                                     <!--
                                         We set attached=true so that the gpg plugin will
-                                        sign all of our assembly artifacts. We handle
-                                        deployment of the artifacts manually because
-                                        maven-deploy-plugin can't upload files to a flat
-                                        S3 directory; it insists on creating a
-                                        long/packagename/and/artifactid/and/version/
-                                        path at the destination before uploading, and we
-                                        don't want that for our files S3. codehaus's
-                                        wagon:upload plugin comes to our rescue.
```

Comment:
I am not familiar with this part. Can you explain why you deleted this paragraph?

Reply:
I just saw this comment which make no sense. We are not using s3 save some files.